.azure-pipelines/all.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         versionSpec: $(PYTHON_VERSION)
     - script: pip install -e .[tests]
       displayName: Install test dependencies
-    - script: npm install -C @openapitools/openapi-generator-cli
+    - script: npm install @openapitools/openapi-generator-cli
```

Comment:
maybe as `-g/--global`?

Reply:
I tried using the -g flag, but then I got this error: 

```
2019-12-09T17:01:11.2597281Z npm ERR! The operation was rejected by your operating system.
2019-12-09T17:01:11.2597549Z npm ERR! It is likely you do not have the permissions to access this file as the current user
2019-12-09T17:01:11.2597741Z npm ERR! 
2019-12-09T17:01:11.2598381Z npm ERR! If you believe this might be a permissions issue, please double-check the
2019-12-09T17:01:11.2598774Z npm ERR! permissions of the file and its containing directories, or try running
2019-12-09T17:01:11.2599309Z npm ERR! the command again as root/Administrator.
```
It works if I use the command with sudo: 
```
sudo npm install -g @openapitools/openapi-generator-cli
```
Nick and I discussed it, and we are not sure about using sudo in this command; what are your thoughts?

Reply:
As a note, the current commit on this branch uses sudo -- I just wanted to show how the tests pass with it.  I can change it if we decide not to use sudo.